start_livia.sh

```
@@ -0,0 +1,55 @@
+echo "📦 Verificando dependências..."
+$PYTHON_CMD -c "import aiohttp" 2>/dev/null || {
+    echo "📦 Instalando aiohttp..."
+    pip3 install aiohttp
+}
+
+# Verificar variáveis de ambiente
+echo "🔐 Verificando variáveis de ambiente..."
+if [ -z "$SLACK_BOT_TOKEN" ] || [ -z "$SLACK_APP_TOKEN" ] || [ -z "$OPENAI_API_KEY" ]; then
+    echo "⚠️  Carregando variáveis de ambiente do .env..."
+    if [ -f ".env" ]; then
+        export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)